src/tests.builds

```diff
@@ -5,7 +5,7 @@
   <Import Project="$(ToolsDir)ConstructSharedFx.targets" Condition="Exists('$(ToolsDir)ConstructSharedFx.targets')" />
 
   <ItemGroup>
-    <Project Include="$(MSBuildThisFileDirectory)*\tests\**\*.csproj" Exclude="@(TestProjectExclusions)">
+    <Project Include="$(MSBuildThisFileDirectory)*\tests\**\*.Tests.csproj" Exclude="@(TestProjectExclusions)">
```

Comment:
I think it is reasonable as well. Can you update the project-guidelines.md file under test section to call out test projects need to end in "Tests".